.github/workflows/release.yaml

```diff
@@ -0,0 +1,79 @@
+name: Release to latest/edge
```

Comment:
This looks great! Thanks for doing this Paulo! Quick question! Should this be merged after legacy relations are finished?